cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/transform/LogParserTransform.java

```diff
@@ -84,8 +84,10 @@
   private static final int CLF_REGEX_LENGTH = 9;
   private static final String S3_LOG = "S3";
   private static final String CLF_LOG = "CLF";
+  private static final String CLOUDFRONT_LOG = "Cloudfront";
   private final LogParserConfig config;
   private final SimpleDateFormat sdf = new SimpleDateFormat("dd/MMM/yyyy:HH:mm:ss Z");
```

Comment:
nit: if the next one is ``sdfCloudFront``, what is this one for? Can you qualify the name?